lib/server/controllers/tags.controller.js

```diff
@@ -41,7 +41,17 @@ const createTag = async (request, response, next) => {
     if (!value) {
         return;
     }
-
+    if ((value?.body?.email || value?.body?.mattermost) &&
+            !request?.session?.access?.includes('admin')) {
+        response.status(403).json({ errors: [
+            {
+                status: '403',
+                title: 'Forbidden',
```

Comment:
This created a bit of a problem recently as the error message was not very informative. I would suggest we give a bite more information. Something in line with
`Non-Admin users are not allowed to provide email and mattermost details for tags"